packages/ui/src/components/DateInput/__stories__/Range.stories.tsx

```diff
@@ -0,0 +1,17 @@
+import { Template } from './Template'
+
+export const Range = Template.bind({})
```

Comment:
Could you make this story like controlled so we can test the component on storybook? I know designers will want to test it haha